redaxo/src/core/lib/util/i18n.php

```diff
@@ -47,7 +47,10 @@ public static function setLocale($locale, $phpSetLocale = true)
 
         if ($phpSetLocale) {
             [$lang, $country] = explode('_', self::getLocale(), 2);
-            Locale::setDefault($lang.'-'.strtoupper($country));
+
+            if (!rex::isSetup() || class_exists(Locale::class)) {
```

Comment:
würde hier nicht einfach 

```suggestion
            if (class_exists(Locale::class)) {
```

reichen?

Reply:
Nach dem Setup erwarten wir die Klasse und es wäre ein Fehler, wenn sie nicht existieren würde. Daher meine Überlegung, wenn kein Setup, die Zeile zu erzwingen.

Reply:
hm setup-assumptions in anderen klassen fänd ich falsch. das hätten wir sonst an vielen stellen auch, oder?

wenn klassen zur setup zeit da waren und auf einmal verschwinden könnten, hätten wir viele andere probleme?

Reply:
> hm setup-assumptions in anderen klassen fänd ich falsch.

Also mein Gedanke war: Normalerweise würde ich hier gar keinen Check haben wollen, sondern direkt die Klasse nutzen, so wie an allen anderen Stellen wir auch alle erforderlichen Klassen normal direkt nutzen.
Nur im Setup, da ein class_exists drum rum, als Sonderfall. 

Aber es ist mir nicht wichtig, dann übernehme ich deinen Wunsch.